contracts/veLON.sol

```diff
@@ -500,7 +529,11 @@ contract veLON is IveLON, ERC721, Ownable, ReentrancyGuard {
 
             // update ts and block (approximately block number)
             poolLastPoint.ts = nextPointTs;
-            poolLastPoint.blk = storedPoolLastPoint.blk + (blockRate * (nextPointTs - storedPoolLastPoint.ts)) / MULTIPLIER;
+            if (blockRate > 0) {
+                poolLastPoint.blk = storedPoolLastPoint.blk + ((nextPointTs - storedPoolLastPoint.ts) * MULTIPLIER) / blockRate;
```

Comment:
In this way, `MULTIPLIER` may be dropped.

Reply:
remove in https://github.com/consenlabs/tokenlon-contracts/pull/81/commits/ef590c464974ef594ff79e9bae2d6d986f5b114d